lib/jnpr/jsnapy/check.py

```diff
@@ -251,6 +251,8 @@ def expression_builder(self, sub_expr, parent_op=None, **kwargs):
                 self.expression_evaluator(elem,**kwargs)
                 res = None
                 #this should be guaranteed by the operator function, never use try-catch here
+                if not kwargs['op'].test_details[kwargs['teston']]:
```

Comment:
What all condition can trigger this not condition? Shouldn't we put a debug message before doing continue?